Goobi/plugins/opac/PicaPlugin/org/goobi/production/plugin/CataloguePlugin/PicaPlugin/Catalogue.java

```diff
@@ -281,4 +283,8 @@ String getUncf() {
 		return ucnf;
 	}
 
+	Map<String,ResolveRule> getResolveRules() {
```

Comment:
Methods without a `private`, `protected` or `public` I read them every time as `public` :( Add a Javadoc comment if needed (method to complex, made more things as the name suggest, ...).